### ISSUE TITLE
onchangeの仕様が異なる

### ISSUE THREAD

**Issue by satakagi** [chirimen-oh/node-web-gpio#33]
CHIRIMEN for Raspberry Pi等のonchangeは　コールバック関数に1/0の数値が返ってくるがこちらはオブヘクトが返り、その.valueに1/0が返る仕様ですね。　どちらが良いかと言われると・・ですが、仕様はどちらかに合わせる必要があると思います。

**Comment by kou029w** [comment]
そうですね。現状 http://browserobo.github.io/WebGPIO/ 時点の仕様をリファレンスとしており、{value, port}が得られるような振る舞いになっています。
